postcode_lookup/template_sorter.py

```diff
@@ -217,18 +247,50 @@ def __init__(
             "timetable": self.timetable,
         }
 
-        enabled_sections = [BallotSection]
-        if not self.all_cancelled:
-            enabled_sections.append(RegistrationDateSection)
+        enabled_sections = [BallotSection(**section_kwargs)]
```

Comment:
Previously we were making a list of classes and then constructing them all into objects at the end with the same kwargs.

I'vw switched to constructing objects as I go. This allows me to pass in custom timetable into each registration section instead of having on global one.